src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -40,37 +41,35 @@
 import net.sf.jabref.logic.labelpattern.DatabaseLabelPattern;
 import net.sf.jabref.logic.util.strings.StringUtil;
 import net.sf.jabref.model.database.BibDatabaseMode;
-import net.sf.jabref.sql.DBStrings;
 
+import com.google.common.eventbus.EventBus;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
 public class MetaData implements Iterable<String> {
 
     private static final Log LOGGER = LogFactory.getLog(MetaData.class);
     public static final String META_FLAG = "jabref-meta: ";
-    private static final String SAVE_ORDER_CONFIG = "saveOrderConfig";
+    public static final String SAVE_ORDER_CONFIG = "saveOrderConfig";
 
-    private static final String SAVE_ACTIONS = "saveActions";
-    private static final String PREFIX_KEYPATTERN = "keypattern_";
-    private static final String KEYPATTERNDEFAULT = "keypatterndefault";
-    private static final String DATABASE_TYPE = "databaseType";
+    public static final String SAVE_ACTIONS = "saveActions";
```

Comment:
Here you can see the remote structure of the metadata table.

![md_table](https://cloud.githubusercontent.com/assets/15609468/16564943/83bb95c4-420a-11e6-9f86-c7819a79ea48.png)

To insert and compare the data I need those attributes (see https://git.io/vKJbl).

Using the `getAsStringMap` method makes it even worse, as I have to parse the String manually. Here is the return value of `getAsStringMap` for the data above:
```
{databaseType=biblatex;, saveActions=enabled;
author[capitalize,html_to_latex]
title[title_case];, saveOrderConfig=specified;author;false;title;false;year;true;}
```
It returns similar result as `Map.toString()`...